gridstatus/eia.py

```diff
@@ -734,20 +753,62 @@ def _handle_fuel_type_data(df):
 
     df["MW"] = df["MW"].astype(float)
 
+    # The raw data will sometimes have case-insensitive duplicates
+    # (e.g. "Pumped Storage","Pumped storage"). We can handle that through the pivot
+    # table by summing the duplicates.
+    df["type-name"] = df["type-name"].str.lower()
+
     # pivot on type
     df = df.pivot_table(
         index=["Interval Start", "Interval End", "Respondent", "Respondent Name"],
         columns="type-name",
         values="MW",
+        aggfunc="sum",
```

Comment:
This assumes that "Pumped storage" and "Pumped Storage" can be added together if they are both present.